zulip_bots/zulip_bots/bots/monkeytestit/monkeytestit.py

```diff
@@ -0,0 +1,52 @@
+import logging
+from typing import Dict, Any
+
+from zulip_bots.bots.monkeytestit.libraries import parse
```

Comment:
We usually abbreviate the "libraries" directories as "lib".

Reply:
I don't quite catch that, sorry. Should I rename the folder `libraries` into `lib`, or should I do something like

```
from zulip_bots.bots.monkeytestit import libraries as lib
from lib import parse
```

Reply:
Just rename the directory.

Reply:
Okay

Reply:
`git add somefile` produces something

```
The following paths are ignored by one of your .gitignore files:
zulip_bots/zulip_bots/bots/monkeytestit/lib/extract.py
Use -f if you really want to add them.
```
---

I followed [this directory structure](https://www.zulipchat.com/api/writing-bots#adding-a-bot-to-zulip) so should be fine with `libraries`?

Reply:
I'm not sure what's happening with your `.gitignore` files here.

Reply:
One thing to know is that `git mv` can be useful when you have a PR and get feedback on directory names.  If you've done `git add libraries/foo.py` and `git add libraries/bar.py`, you can do `git mv libraries lib` to shorten the folder name.

Reply:
I think it's ignoring `lib` because it's in https://github.com/zulip/python-zulip-api/blob/master/.gitignore